extension.js

```diff
@@ -188,6 +197,7 @@ const SensorsMenuButton = new Lang.Class({
             // Configure as main sensor and set panel string
             item.setMainSensor();
             this.statusLabel.set_text(item.getPanelString());
+            //this.statusLabel.set_text(allCoreTemps);
```

Comment:
Doesn't need to be, that was my first crack at it, before I decided to make it configurable I just hacked all the cores in for my personal usage.